src/sentry/eventstream/snuba.py

```diff
@@ -288,9 +286,9 @@ def end_unmerge(self, state: Mapping[str, Any]) -> None:
             state_copy["project_id"], "end_unmerge", extra_data=(state_copy,), asynchronous=False
         )
 
-    def start_delete_tag(self, project_id: int, tag: str) -> Mapping[str, Any] | None:
+    def start_delete_tag(self, project_id: int, tag: str) -> Mapping[str, Any]:
         if not tag:
-            return None
```

Comment:
these `None`s (same as above) would immediately crash in the `end_*` methods so I changed this to an error and removed the nullability